schema/project-level/codelists/metricID.csv

```diff
@@ -0,0 +1,3 @@
+Code,Title,Description
+financialProgress,Financial Progress,"The planned or actual financial progress for the project. For example, the percentage of spend against the project budget."
+physicalProgress,Physical Progress,"The planned or actual physical progress for a project. For example, the percentage of completion of the physical works."
```

Comment:
In codelists like documentType we use "Sentence case" instead of "Title Case" for the Title column.

Reply:
Fixed in https://github.com/open-contracting/infrastructure/pull/282/commits/683ef4da0c2d20dfc26e94784e7db0d2fcfeec61